### PR TITLE
Add "What is "publish size" vs "install size"?" section

### PR DESCRIPTION
**Pull request by avdeev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 Little utility to ease troubleshooting why installing npm dependencies takes too long.
 
+## What is "publish size" vs "install size"?
+
+The "publish size" is the size of the source code published to npm. This number is easy to detect and is typically very small.
```
Comment (antonk52) [inline README.md]:
```suggestion
The "publish size" is the size of the source code published to npm. This number is easy to detect and should be pretty small.
```

**Review by antonk52 — COMMENTED** [review]
Awesome! Thank you, I have a single comment on phrasing, once it is fixed, let's merge 🤝

**Comment by avdeev** [comment]
@antonk52 Sure :) Updated!

**Comment by antonk52** [comment]
Nice one! Thank you 🥳

**Pull request merged**